docs/tutorials/models.ipynb

```diff
@@ -15,9 +15,11 @@
     "\n",
```

Comment:
<p>There is a property <code>is_norm_spectral_model</code>  that you can use: https://github.com/gammapy/gammapy/blob/master/gammapy/modeling/models/spectral.py#L55</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Yeah I wanted to use it, but ran into the (probably stupid) issue that some models need parameters upon initialisation, eg `PiecewiseNormSpectralModel`, so found it difficult to access the `is_norm_spectral_model` property inside the loop

Reply:
Indeed this doesn't work...however we could transform `is_norm_spectral_model` into a "classproperty" (https://docs.astropy.org/en/stable/api/astropy.utils.decorators.classproperty.html), this avoids the need for initialisation...

Reply:
Ah I did not know this "classproperty"  option. Nice!
Maybe we can merge this PR now (nice to have this merged before the modelling tutorial) and I add this is a follow-up PR?

Reply:
Yes, sure...I‘ll go ahead and merge now...